### DISCUSSION TITLE
How to fill NaN with the previous value?

### DISCUSSION THREAD

**Discussion by iam-yan** [opensource9ja/danfojs#409] [Q&A] [answered]
In pandas, when filling NaN values, you can pick a method to calculate the value for filling. e.g. `fillna(method='bfill')`
How can I achieve this in Danfo?

**Comment by risenW** [comment, marked as answer]
@iam-yan This feature is not yet supported

**Reply by iam-yan** [reply]
I c. Thanks. :)

**Comment by danmaty** [comment]
Hi,
Has this been implemented since?
